test/utils/test_interp.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import pytest
+import torch
+
+from earth2studio.utils.interp import LatLonInterpolation
+
+
+@pytest.mark.parametrize("device", ["cpu", "cuda:0"])
+@pytest.mark.parametrize("input_type", ["zeros", "random", "gradient"])
+def test_interpolation(device, input_type):
```

Comment:
Any change you can implement at unit test here that checks the qualitative value of the output.

So have maybe you have a 2x2 reference in a [0,1] box with some fixed values at the corners. Then you have a few points at like [0.25,0.25] [0.25,0.5] [0.5.0.25] and [0.5 0.5].

And confirm the outputs are analytically what one would expect

Reply:
Added in https://github.com/NVIDIA/earth2studio/pull/159/commits/4df40a1f4ca4dba0c8da60f9d9fc6a6544546754.